lib/services/workflow-api-service.js

```diff
@@ -93,6 +95,7 @@ function workflowApiServiceFactory(
             });
         })
         .then(function(graph) {
+            sanitizer.decrypt(graph);
```

Comment:
I'm not quite sure what impact this has other than sending unencrypted values back to the client?